FritoMenu/Models/Order/Order.swift

```diff
@@ -17,20 +17,22 @@ struct Order: Equatable, Identifiable {
     var id: UUID
 
     /// Items.
-    var items: IdentifiedArrayOf<OrderItem>
+    var items: [OrderItem: Int]
```

Comment:
No.

What you should have done, is make a computed property on `Order` that provides this counted list.

And then consume this counted list in the `OrderView` instead of the raw `Order`.